numpyro/examples/datasets.py

```diff
@@ -70,6 +76,37 @@
     ["https://archive.ics.uci.edu/ml/machine-learning-databases/00280/HIGGS.csv.gz"],
 )
 
+LR_BANANA = dset(
+    "lr_banana", ["https://github.com/pyro-ppl/datasets/raw/master/benchmarks.mat"]
```

Comment:
The filename benchmarks is confusing. Could you rename the file? Also, if mat is not a convenient format (requires scipy) then it is better to store it as npz file

Reply:
I'll fix the names and convert them to NumPy format.

Reply:
On second thought, I removed it. I'm not sure I'll add the example to numpyro, and if I do, I'll fix the problem then.